api/clients/v2/examples/example_relay_retrieval_test.go

```diff
@@ -0,0 +1,86 @@
+package examples
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/status-im/keycard-go/hexutils"
+)
+
+// This example demonstrates how to use the RelayPayloadRetriever to retrieve a payload from EigenDA, running on
+// holesky testnet
+func Example_relayPayloadRetrieval() {
+	// You must provide a private key that either has a testnet reservation, or you must configure on-demand payments
+	// by sending funds to the payment vault.
+	privateKey := ""
+
+	// Create a payload disperser and disperse a sample payload to EigenDA
+	// This will be the payload we will later retrieve
+	payloadDisperser, err := createPayloadDisperser(privateKey)
+	if err != nil {
+		panic(fmt.Sprintf("create payload disperser: %v", err))
+	}
+	defer payloadDisperser.Close()
+
+	payload, err := createRandomPayload(4 * 1024) // (4KB of random data)
+	if err != nil {
+		panic(fmt.Sprintf("create random payload: %v", err))
+	}
+
+	dispersalCtx, dispersalCancel := context.WithTimeout(context.Background(), 1*time.Minute)
```

Comment:
```suggestion
	dispersalCtx, dispersalCancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Reply:
Otherwise gives the impression this actually requires some processing and is a long blocking call

Reply:
[313ed140](https://github.com/Layr-Labs/eigenda/pull/1454/commits/313ed140bed4634181fbc7e636cb0f2ee9ddf7c0)

I've seen dispersal take 20 seconds in practice, though it's not the standard delay

Reply:
I actually realized that the timeouts being set for the payload clients are redundant, since they already have configurable internal timeouts.

I removed the redundancy in [b410361d](https://github.com/Layr-Labs/eigenda/pull/1454/commits/b410361d34476a79e48278eec5ba6ce89c8f63d1)